Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 Language Features:
  * General: Support ``ContractName.functionName`` for ``abi.encodeCall``, in addition to external function pointers.
+ * General: Adds equality-comparison operators for external function types.
```

Comment:
```suggestion
 * General: Add equality-comparison operators for external function types.
```

Since you now quickly changed it once, maybe also loose the ``s`` to be consistent with the style of the other entries, then we can finally merge :-).